debian/clearwater-etcd.prerm

```diff
@@ -64,6 +64,9 @@ case "$1" in
         # Make sure the process is stopped
         service clearwater-etcd stop
 
+        # Clean up old pidfile
+        rm -f /var/run/$NAME.pid
```

Comment:
Sure - and now that it's moved, having an old PID lying around in /var/run/clearwater-etcd.pid is just confusing, so I've added code to clean that up on upgrade. I'll make the comment more explicit.